tests/components/humidifier/test_recorder.py

```diff
@@ -37,7 +37,10 @@ async def test_exclude_attributes(recorder_mock: Recorder, hass: HomeAssistant)
     )
     assert len(states) >= 1
     for entity_states in states.values():
-        for state in entity_states:
+        for state in filter(
+            lambda state: split_entity_id(state.entity_id)[0] == humidifier.DOMAIN,
```

Comment:
Why not use a list comprehension?

Reply:
Because we would build the list and then throw it away. It's still probably faster to o do that though since filter has the cost of the lambda.

Reply:
I'll change it to a genexpr